code/modules/mob/living/carbon/life.dm

```diff
@@ -455,4 +455,33 @@
 		death()
 		var/obj/item/organ/brain/B = getorganslot(ORGAN_SLOT_BRAIN)
 		if(B)
-			B.damaged_brain = TRUE
\ No newline at end of file
+			B.damaged_brain = TRUE
+
+
+/////////////////////////////////////
+//MONKEYS WITH TOO MUCH CHOLOESTROL//
+/////////////////////////////////////
+
+/mob/living/carbon/proc/can_heartattack()
+	CHECK_DNA_AND_SPECIES(src)

```

Comment:
this will return null if the monkey has no species, which is doesn't, so can_heartattack will always be no

Reply:
Thanks